pkg/autodctest/startupscript.go

```diff
@@ -139,7 +139,7 @@ prepare_necoapps()
 mkdir -p ${GOPATH}/src/github.com/cybozu-go &&
 cd ${GOPATH}/src/github.com/cybozu-go &&
 gcloud secrets versions access latest --secret="%s" > cybozu_private_repo_read_pat &&
-git clone --depth 1 -b %s https://cybozu-neco:$(cat cybozu_private_repo_read_pat)@github.com/cybozu-go/neco-apps &&
+git clone --depth 1 -b %s https://cybozu-neco:$(cat cybozu_private_repo_read_pat)@github.com/cybozu-private/neco-apps &&
```

Comment:
In this section, cybozu-<b>private</b>/neco-apps is still cloned into .../cybozu-<b>go</b>/neco-apps . I think it is a little confusing.

Reply:
Fixed the clone destination for neco-apps.
I have also confirmed that the operation is not affected.